test/methods.test.ts

```diff
@@ -0,0 +1,8 @@
+test('has exposed some desired methods', () => {
+  expect({
+    numberVal: .31,
+  }).toEqual({
+    numberVal: expect.expect('toBeCloseTo', 0.3, 1),
```

Comment:
question: do we want:
```Typescript
numberVal: expect.expect('toBeCloseTo', number, number);
```
Or:
```Typescript
numberVal: expect.toBeCloseTo(number, number);
```

Reply:
I'm not sure how it would be adding each of these to the root level, but I guess we could do it that way.